public/css/policy.css

```diff
@@ -0,0 +1,32 @@
+article, p{
```

Comment:
try to avoid adding styles directly to elements.
This can lead to issues in other pages